src/app/modules/main/profile_section/wallet/accounts/view.nim

```diff
@@ -62,3 +63,17 @@ QtObject:
   proc setKeyPairModelItems*(self: View, items: seq[KeyPairItem]) =
     self.keyPairModel.setItems(items)
     self.keyPairModelChanged()
+
+  proc includeWatchOnlyAccountChanged*(self: View) {.signal.}
+  proc getIncludeWatchOnlyAccount(self: View): bool {.slot.} =
+    return self.includeWatchOnlyAccount
```

Comment:
In this case having `includeWatchOnlyAccount` as a member of `View` and maintaining the state if watch only accounts are included or not is unnecessary when we already have our source of truth in settings, maybe just call `self.delegate.isIncludeWatchOnlyAccount()` from here. Up to you.

Reply:
id like to keep it this way then :)